extensions/functions_boolean.yaml

```
@@ -110,3 +110,29 @@ scalar_functions:
           - value: boolean?
             name: a
         return: boolean?
+  - 
+    name: "bool_and"
+    description: >
+      Aggregate boolean values and returns a boolean value.
+      If all of the values in the aggregation are true, function returns TRUE,
+      else it returns FALSE if any of the returned values are false. NULL values
+      are ignored by this function.
+    impls:
+      - args:
+          - value: boolean
+            name: x
+        nullability: DECLARED_OUTPUT
+        return: boolean
+  - 
+    name: "bool_or"
+    description: >
+      Aggregate boolean values and returns a boolean value.
+      If any value in the aggregation is true, function returns TRUE,
+      else it returns FALSE if all of the returned values are false. NULL values
+      are ignored by this function.
+    impls:
+      - args:
+          - value: boolean
+            name: x
+        nullability: DECLARED_OUTPUT
+        return: boolean
```

At this point I might as well just do it like this:

```suggestion
  - 
    name: "bool_and"
    description: >
      Aggregate boolean values and returns a boolean value. If any value in the
      aggregation is false, false is returned. If the input is empty or only
      contains nulls, null is returned. Otherwise, true is returned.
    impls:
      - args:
          - value: boolean
            name: x
        nullability: DECLARED_OUTPUT
        return: boolean?
  - 
    name: "bool_or"
    description: >
      Aggregate boolean values and returns a boolean value. If any value in the
      aggregation is true, true is returned. If the input is empty or only
      contains nulls, null is returned. Otherwise, false is returned.
    impls:
      - args:
          - value: boolean
            name: x
        nullability: DECLARED_OUTPUT
        return: boolean?
```

---

Updated.

---

Thanks for the suggestion.